mtp_noms_ops/templates/security/prisoners-detail.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load security %}
+
+{% block inner_content %}
+
+  <header>
+    {% language_switch %}
+    <h1 class="heading-xlarge">{{ prisoner.prisoner_number}} {{ prisoner.prisoner_name }}
+      <p class="lede">{% trans 'Held at ' %}{{ prisoner.current_prison.name }}</p>
```

Comment:
would be better as:

```html
{% blocktrans with prison=risoner.current_prison.name %}Held at {{prison}}{% endblocktrans %}
```

since trailing space is easy to miss and word order may vary per language